package.json

```diff
@@ -15,7 +15,8 @@
     "compression": "^1.7.1",
     "polka": "^0.3.4",
     "sapper": "^0.12.0",
-    "serve-static": "^1.13.1"
+    "serve-static": "^1.13.1",
```

Comment:
serve-static should be completely unnecessary now, no?

Reply:
whoops, not sure what happened here